src/java/org/apache/cassandra/cql3/constraints/ConstraintFunction.java

```diff
@@ -41,13 +44,22 @@ public abstract class ConstraintFunction
 {
     public static final List<Operator> DEFAULT_FUNCTION_OPERATORS = List.of(EQ, NEQ, GTE, GT, LTE, LT);
 
-    protected final ColumnIdentifier columnName;
+    protected ColumnIdentifier columnName;
     protected final String name;
+    protected final List<String> args;
+    // args as propagated from cql
+    protected final List<String> rawArgs;
 
-    public ConstraintFunction(ColumnIdentifier columnName, String name)
+    public ConstraintFunction(String name, List<String> args)
     {
-        this.columnName = columnName;
         this.name = name;
+        this.rawArgs = args;
+        this.args = unquote(args);
```

Comment:
I want to work with unquoted arguments, quoted (if any) from CQLSH are here for description only